tests/github/repo1/test_checker_no_args.py

```diff
@@ -14,132 +14,132 @@ class TestCheckerNoArgs(Contract):
 
     def test_check_checklist(self, mocker):
         with mocker:
-            mocked_checker = get_mocked_checker()
+            mocked_checker = get_checker()
```

Comment:
Can the variable also be renamed to `checker`? 

Let's also remove `mocked_` from all variables set from a `get_checker()` or `get_repo()` method.

Reply:
done